server/src/test/java/io/spine/server/stand/StandStorageShould.java

```diff
@@ -60,7 +57,7 @@
  * @author Dmytro Dashenkov
  */
 public abstract class StandStorageShould extends RecordStorageShould<AggregateStateId,
-                                                                     RecordStorage<AggregateStateId>> {
+                                                                     StandStorage> {
```

Comment:
I think this is a change, which will may not fit well into the dependent modules (such as `jdbc-storage`).

Reply:
I tested this for `jdbc-storage` and it's okay. For `gae-java` it should be also fine.